### ISSUE TITLE
About normalized weight

### ISSUE THREAD

**Issue by lincoln0211** [jychoi118/P2-weighting#13]
Thanks for your work.

I have one questions for training p2-weighting.
How can I find the normalized weight part in the code like it is in the paper?
Are the results in the paper indicative of unnormalized weights?

Thanks again and look forward to your reply.

**Comment by dolphin0104** [comment]
Same question. 
the results reported in Tables 1, 2, and 3 of the paper obtained using normalized weights? Or are they achievable using unnormalized weights?

**Comment by jychoi118** [comment]
We trained models with unnormalized weights. Increasing the learning rate can be an alternative of using normalized weights.

**Comment by lincoln0211** [comment]
Thank you for your prompt response!
